config/feature.js

```diff
@@ -10,7 +8,9 @@ function getValue(key) {
   return pick(config, key);
 }
 
-const isEnabled = getValue;
+function isEnabled(key) {
+  return pick(config, `features.${key}`);
```

Comment:
Why use `pick` here? `features[key]` should be fine. One thing I really don't like about lodash is it doesn't error on `null`, so `pick(null, ...)` just returns an empty object.